File: django_apollo/blog/types/user_type.py
import graphene
from graphene.contrib.django import DjangoObjectType
from django.contrib.auth.models import User


class UserType(graphene.ObjectType):
    id = graphene.String()
    first_name = graphene.String(description='first name of the user')
    last_name = graphene.String(description='last name of the user')
    email = graphene.String(description='email of the user')
    username = graphene.String(description='username of the user')
    posts = graphene.List('posttype', description='Posts of this user')

File: node/express_graphql/graphql/types/userType.js
import postType from './postType';
import commentType from './commentType';
import models from '../../models';
import { resolver, attributeFields } from 'graphql-sequelize';
import { GraphQLObjectType, GraphQLNonNull, GraphQLList, GraphQLInt, GraphQLString} from 'graphql';
import { _ } from 'underscore';

let userType = new GraphQLObjectType({
  name: 'User',
  description: 'A user',
  fields: () => ({
    id: {
      type: new GraphQLNonNull(GraphQLInt),
      description: 'The id of the user.',
    },
    first_name: {
      type: GraphQLString,
      description: 'The first name of the user.',
    },
    last_name: {
      type: GraphQLString,
      description: 'The last name of the user.',
    },
    username: {
      type: GraphQLString,
      description: 'The username of the user.',
    },

    email: {
      type: GraphQLString,
      description: 'The email of the user.',
    },
  })
});

module.exports = userType;
